.gitlab-ci.yml

```diff
@@ -70,6 +70,7 @@ make-coq-latest:
   script:
     - docker build --pull --build-arg=coq_image="coqorg/${CI_JOB_NAME//-/:}" --build-arg=compiler="${OPAM_SWITCH}" -t "${IMAGE}" .
     - docker push "${IMAGE}"
+    - docker run --mount "type=bind,source=$(pwd),target=/home/coq/mathcomp" -w /home/coq/mathcomp -t "${IMAGE}" /bin/bash --login -c "set -x; cd mathcomp; make test-suite"
```

Comment:
This command works but the place where it is run is suboptimal. More precisely:
It runs `make test-suite` (existing tests + notation test) for each version of Coq, which is very fine, but these tests should ideally be separated from the build job to "separate concerns", so that:

* the other stages don't wait too much to be started (otherwise it would take [42'](https://gitlab.com/math-comp/math-comp/-/jobs/729111262) while it used to take [26'](https://gitlab.com/math-comp/math-comp/-/jobs/730243138)),
* and a failure in this new notation test won't prevent other tests to be run.

However, the change I suggest (cf. my main review post) should not prevent running these tests in the nightly build, so that we can be aware of any unexpected notation change from coq.dev.